torchvision/__init__.py

```diff
@@ -1,16 +1,24 @@
+from modulefinder import Module
 import os
 import warnings
 
 import torch
 from torchvision import datasets, io, models, ops, transforms, utils
 
-from .extension import _HAS_OPS
+from .extension import _HAS_OPS, _load_library
 
 try:
     from .version import __version__  # noqa: F401
 except ImportError:
     pass
 
+try:
+    _load_library("Decoder")
```

Comment:
Do you know why we didn't have this previously?

Reply:
I see you moved it from io

Reply:
yup